package.json

```diff
@@ -18,8 +18,9 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "lit-element": "^2.4.0",
-    "chalk": "^4.1.0"
+    "@alaskaairux/orion-icons": "^2.1.5",
```

Comment:
Why are you making a dependency on a very old version? 

https://github.com/AlaskaAirlines/Icons